openfecwebapp/templates/advanced.html

```diff
@@ -97,6 +106,10 @@ <h1 class="icon-heading__title">{{ title }}</h1>
       <div id="receipts" class="option">
         <h2>Receipts</h2>
         <div class="option__content">
+          <p><span class="term" data-term="receipt">Receipts</span> are anything of value, like money, services or property. Search 
+          candidate and committee receipt data by contributor, amount and date.</p>
```

Comment:
Sorry, another thing I should have caught: do we want to add something like "Receipts include contributions from individuals as well as from other committees" in order to make clear that "individual contributions" are one type of receipt?